od-czego-zaczac/README.md

```diff
@@ -27,11 +27,11 @@ Więcej materiałów znajduje się w poszczególnych rozdziałach i w przygotowa
 ### Pierwsza praca jako tester
 
 Wiesz jak wygląda praca testera?  
-Z czym będziesz musiał się zmierzać każdego dnia?
+Z czym będziesz musiał się mierzyć każdego dnia?
```

Comment:
Jak tak teraz myślę i czytam to poprawnie jest "mierzyć się" :)

Reply:
Agree.
"Się mierzyć" jest chyba bardziej melodyjne, jednak w zestawieniu z poprzednim zdaniem "mierzyć się" brzmi zdecydowanie lepiej.
Changed.